cmd/manager/main.go

```diff
@@ -126,13 +128,20 @@ func main() {
 	// work
 	stopCh := signals.SetupSignalHandler()
 
+	client1, err := client.New(cfg, client.Options{})
```

Comment:
I think we can do better than `client1` `accountWatcherClient` maybe?

Reply:
Yup makes sense to me.